travis/pmd-ruleset.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+
+<ruleset name="Custom Rules"
+		 xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+
+	<description>
+		RuneLite client PMD ruleset
+	</description>
+
+	<!--<rule ref="category/java/bestpractices.xml">
+		<exclude name="SwitchStmtsShouldHaveDefault" />
+	</rule>
+	-->
+
+	<!-- Code style rules -->
+	<rule ref="category/java/codestyle.xml">
+		<exclude name="AtLeastOneConstructor"/>
+		<exclude name="CommentDefaultAccessModifier"/>
+		<exclude name="ConfusingTernary"/>
+		<exclude name="ControlStatementBraces"/>
+		<exclude name="DefaultPackage"/>
+		<exclude name="LocalVariableCouldBeFinal"/>
+		<exclude name="LongVariable"/>
+		<exclude name="MethodArgumentCouldBeFinal"/>
+		<exclude name="OnlyOneReturn"/>
+		<exclude name="ShortClassName"/>
+		<exclude name="ShortVariable"/>
+		<exclude name="TooManyStaticImports"/>
+		<exclude name="UnnecessaryFullyQualifiedName"/>
+		<exclude name="UselessParentheses"/>
+		<exclude name="UseUnderscoresInNumericLiterals"/>
+
+		<!-- TODO: reenable later, but cause a lot of noise -->
```

Comment:
I'd like to enable these rules again, but they lead to about 1.7k additional violations currently. Thoughts?

Reply:
These seem reasonable to me.